api-reference/v2/resources/changelog.mdx

```diff
@@ -3,6 +3,12 @@ title: Glide API Changelog
 sidebarTitle: Changelog
 ---
 
+### November 21, 2024
+
+- Introduced `HEAD /tables/{tableID}/rows` endpoint to retrieve the current version of table data via `ETag` header.
```

Comment:
Is this really the version of the rows, or the table itself? I would've expected to see this path just be `/tables/{tableID}`

Reply:
> Is this really the version of the rows, or the table itself?

Yeah, ideally this would be the max version of the specific set of rows returned by the equivalent `GET` request*, not the version of the whole table. But that depends on Larry's work landing. So for now it's the table version and I've opened https://github.com/glideapps/glide/issues/30958. Note that the table version is always greater than or equal to the versions of all its rows, so this is still valuable for preventing data loss, just not as granular for change detection.

> I would've expected to see this path just be `/tables/{tableID}`

We might add that path down the line, but that would also return the table's schema and the `ETag` would also need to cover changes to that.

---

\* Note that this is also why I added the clarification about `limit` to the `GET` endpoint docs.. philosophically, not supplying a limit means "the whole table", so the returned `ETag` would be the table version, even if fewer rows are actually returned at a time due to operational logistics.